mix.exs

```diff
@@ -1,39 +1,56 @@
 defmodule MixTestWatch.Mixfile do
   use Mix.Project
 
+  @source_url "https://github.com/lpil/mix-test.watch"
   @version "1.0.2"
 
   def project do
     [
       app: :mix_test_watch,
       version: @version,
-      elixir: "~> 1.0",
+      elixir: "~> 1.5",
       build_embedded: Mix.env() == :prod,
       start_permanent: Mix.env() == :prod,
       deps: deps(),
+      docs: docs(),
       name: "mix test.watch",
       description: "Automatically run tests when files change",
       package: [
         maintainers: ["Louis Pilfold"],
         licenses: ["MIT"],
-        links: %{"GitHub" => "https://github.com/lpil/mix-test.watch"},
+        links: %{
+          "Changelog" => "https://hexdocs.pm/mix_test_watch/changelog.html",
```

Comment:
Is this the right URL for the changelog? Should it be the file in the repo instead?